allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -207,16 +208,48 @@ def from_data_file(cls, data_file: EyeTrackingFile,
         dilation_frames : int, optional
              See EyeTracking.from_lims
         """
+        def _get_lost_frames(file_path: str) -> List[int]:
+            """
+            Get lost frames from the video metdata json
+            Must subtract one since the json starts indexing at 1
+
+            Parameters
+            ----------
+            file_path: str
+                Path to the metadat json
+
+            Returns
+            -------
+                indices of lost frames
+            """
+            with open(file_path, 'r') as f:
+                video_metadata = json.load(f)
+
+            lost_frames = video_metadata['RecordingReport']['LostFrames']
+
+            if lost_frames:
+                return [int(f)-1 for f in lost_frames]
+            else:
+                return []
+
         cls._logger.info(f"Getting eye_tracking_data with "
                          f"'z_threshold={z_threshold}', "
                          f"'dilation_frames={dilation_frames}'")
 
         sync_path = Path(sync_file.filepath)
 
+        if session_data:
+            metadata_json = session_data['raw_eye_tracking_video_meta_data']
+            lost_frames = _get_lost_frames(metadata_json)
+        else:
+            lost_frames = []
+
         frame_times = sync_utilities.get_synchronized_frame_times(
             session_sync_file=sync_path,
-            sync_line_label_keys=Dataset.EYE_TRACKING_KEYS,
-            trim_after_spike=False)
+            sync_line_label_keys='eye_cam_exposing',
```

Comment:
Should this key be added to `Dataset` to keep with the current pattern?

Reply:
I added it to Dataset.EYE_TRACKING_KEYS. Would that work?

Reply:
`get_edges` says 
>         keys : These will be checked in sequence. Timestamps will be returned
            for the first which is present in the line labels

So if none of the other keys in `EYE_TRACKING_KEYS` is found, then this is fine